business_requirement/__openerp__.py

```diff
@@ -5,7 +5,7 @@
     'name': 'Business Requirement',
     'category': 'Business Requirements Management',
     'summary': 'Business Requirement',
-    'version': '8.0.5.0.0',
+    'version': '8.0.5.0.1',
```

Comment:
Since you added new fields, the model needs to be upgraded, right? In that case, version should be 8.0.5.1.0 if I'm correct